src/test/java/core/basesyntax/operationswithfile/FileWriterImplTest.java

```diff
@@ -0,0 +1,38 @@
+package core.basesyntax.operationswithfile;
+
+import static org.junit.Assert.assertEquals;
+
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.HashMap;
+import java.util.Map;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class FileWriterImplTest {
+    private static final String TRUE_PATH = "src/test/java/resources/file.csv";
+    private static final String FALSE_PATH = "srv/test/java/core/basesyntax/resources/newFile.csv";
+    private static final Map<String, Integer> balance = new HashMap<>();
+    private static final FileWriter fileWriter = new FileWriterImpl();
+
+    @BeforeClass
+    public static void addToBalanceMap() {
+        balance.put("banana", 50);
+        balance.put("apple",20);
+    }
+
+    @Test
+    public void testWriteToFileWithTruePath_ok() {
+        fileWriter.getNewFile(balance,TRUE_PATH);
+        boolean actual = false;
```

Comment:
let's compare as well that is the way we expect it to be. Read info from file and compare with expected or at least the file content size is correct

Reply:
created check for file length